tools/eslint-rules/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bitovi/eslint-plugin",
-  "version": "2.0.0",
+  "version": "1.0.1",
```

Comment:
Is this going to be auto bumped?

Reply:
yea, this PR won't be released directly, we'll release through github actions instead